test/extended/marketplace/marketplace_lables.go

```diff
@@ -0,0 +1,94 @@
+package marketplace
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Feature:Marketplace] Marketplace resources with labels provider displayName", func() {
+
+	defer g.GinkgoRecover()
+
+	var (
+		oc            = exutil.NewCLI("marketplace", exutil.KubeConfigPath())
+		allNs         = "openshift-operators"
+		marketplaceNs = "openshift-marketplace"
+
+		//buildPruningBaseDir = exutil.FixturePath("testdata", "marketplace")
```

Comment:
Should this commented out code be removed?

Reply:
will remove